deploy/templates/azuredeploy.simulation.json

```diff
@@ -383,6 +386,492 @@
                 "[variables('vmResourceId')]"
             ]
         },
+        {
```

Comment:
Can we move this into its own arm template?  Like azuredeploy.diagnostics.json`?

Reply:
I would do this in separate PR, I would also push the changes to dashboard along with it.